src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -259,6 +259,10 @@
     public static final String REPAIRJOB_MAX_KEYS_SCANNED_PER_SEC = "repairjob.max.keys.scanned.per.sec";
     public static final String PRUNEJOB_MAX_KEYS_SCANNED_PER_SEC = "prunejob.max.keys.scanned.per.sec";
     public static final String SLOP_PURGEJOB_MAX_KEYS_SCANNED_PER_SEC = "slop.purgejob.max.keys.scanned.per.sec";
+    public static final String ENABLE_NODE_ID_DETECTION = "enable.node.id.detection";
```

Comment:
> node.id.implementation is useful for Testing.
I suspect it isn't, because you haven't used it that way.

> node.id.host.types are useful when you only want host name or FQDN matching.
in other words, not useful at all.